tests/e2e/visual/contacts/list-view-login-visual.wdio-spec.js

```diff
@@ -0,0 +1,134 @@
+const commonPage = require('@page-objects/default/common/common.wdio.page');
+const contactPage = require('@page-objects/default/contacts/contacts.wdio.page');
+const loginPage = require('@page-objects/default/login/login.wdio.page');
+const dataFactory = require('@factories/cht/generate');
+const reportsPage = require('@page-objects/default/reports/reports.wdio.page');
+const utils = require('@utils');
+
+const { resizeWindowForScreenshots, generateScreenshot } = require('@utils/screenshots');
+
+describe('Contact Page | List View (Access)', () => {
+  const updateRolePermissions = async (roleValue, addPermissions, removePermissions = []) => {
+    const roles = [roleValue];
+    const settings = await utils.getSettings();
+    settings.roles[roleValue] = { offline: true };
+    const permissions = await utils.getUpdatedPermissions(roles, addPermissions, removePermissions);
+    await utils.updateSettings(
+      { roles: settings.roles, permissions },
+      { revert: true, ignoreReload: true, refresh: true, sync: true }
+    );
+  };
+
+  const docs = dataFactory.createHierarchy({
+    name: 'Janet Mwangi',
+    user: true,
+    nbrClinics: 10,
+    nbrPersons: 4,
+    useRealNames: true,
+  });
+
+  before(async () => {
+    await resizeWindowForScreenshots();
+    await utils.saveDocs([...docs.places, ...docs.clinics, ...docs.persons, ...docs.reports]);
+    await utils.createUsers([docs.user]);
+  });
+
+  after(async () => {
+    await utils.deleteUsers([docs.user]);
+    await utils.revertDb([/^form:/], true);
+  });
+
+  beforeEach(async () => {
+    await loginPage.login(docs.user);
+  });
+
+  afterEach(async () => {
+    await commonPage.logout();
+  });
+
+  describe('Log in', () => {
+    it('having can_view_contact and can_view_contacts_tab permissions, ' +
+      'should show contacts page as tab, hidden from menu option', async () => {
```

Comment:
Usually start with the "expected" then the "condition"
```suggestion
    it('should show contacts page tab when can_view_contact and can_view_contacts_tab permissions are enabled', async () => {
```